src/CMakeLists.txt

```diff
@@ -446,7 +446,7 @@ if(WITH_JAEGER)
   add_dependencies(common-objs ${jaeger_base})
   target_link_libraries(jaeger-base INTERFACE
     yaml-cpp::yaml-cpp
-    opentracing::libopentracing
+    opentelemetry::libopentelemetry
     thrift::libthrift
     jaegertracing::libjaegertracing)
```

Comment:
I think that `yaml-cpp` and `jaegertracing` libs aren't dependencies of opentelemetry.